app/_src/mesh/features/rbac.md

```diff
@@ -154,6 +154,13 @@ rules:
 {% endnavtab %}
 {% endnavtabs %}
 
+{:.important}
+> **Important:** Granting access actions without binding them to specific resource types provides full access, including to Secrets and GlobalSecrets, posing security risks.
+>
```

Comment:
```suggestion
> <br><br>
```
These callout blocks don't respect paragraph breaks, so we have to add them manually.